src/v2/api/group/GroupRepository.ts

```diff
@@ -1,190 +1,72 @@
 import { Injectable } from '@nestjs/common';
 import { PrismaService } from '../../database/PrismaService';
 import { Prisma } from '@prisma/client';
-import { QueryAllDTO } from '../../utils/QueryAllDTO';
-import { DatabaseUtils } from '../utils/DatabaseUtils';
-import { UpdateGroupDTO } from './dto/UpdateGroupDTO';
 
 @Injectable()
 export class GroupRepository {
+
+  private include = {
```

Comment:
i don't really understand the purpose of these includes, because you haven't used it anywhere

Reply:
solved in ece85c0